nbs/docs/models/Holt.ipynb

```diff
@@ -112,7 +112,7 @@
     "\n",
     "For each method there exist two models: one with additive errors and one with multiplicative errors. The point forecasts produced by the models are identical if they use the same smoothing parameter values. They will, however, generate different prediction intervals.\n",
     "\n",
-    "To distinguish between a model with additive errors and one with multiplicative errors. We label each state space model as ETS( .,.,.) for (Error, Trend, Seasonal). This label can also be thought of as ExponenTial Smoothing. Using the same notation as in Table 7.5, the possibilities for each component are: Error  ={A,M } , Trend  ={ N,A,A d} and Seasonal  ={ N,A,M }\n",
+    "To distinguish between a model with additive errors and one with multiplicative errors. We label each state space model as ETS( .,.,.) for (Error, Trend, Seasonal). This label can also be thought of as ExponenTial Smoothing. Using the same notation as in Table 7.5, the possibilities for each component are: Error  =\\\\{A,M } , Trend  =\\\\{ N,A,A d} and Seasonal  =\\\\{ N,A,M }\n",
```

Comment:
This one looks wrong 
![image](https://github.com/Nixtla/statsforecast/assets/8473587/b2b60e88-3016-4760-8987-6bb79a02daaf)

Reply:
how does this look?
<img width="1153" alt="image" src="https://github.com/Nixtla/statsforecast/assets/55263191/fa3333ea-58d6-44c0-8a34-d9b996d3d9a9">

Reply:
Great. Thanks!